src/pkg/projects/depproj.targets

```diff
@@ -0,0 +1,109 @@
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Can you rename this file to something else?

Reply:
Sure, but what's bad about the current name so I don't just give it another bad one? 😄

Reply:
Not specific enough? Maybe `framework.dependency.targets`, `framework.packaging.targets`? Shared prefix also seems nice to tell what they're for at a glance.

Reply:
depproj.targets overlaps with a target named the same thing elsewhere that implements the basic depproj functionality.

Both of your suggestions seem reasonable.